docs/modules/control/move_to_a_pose_control/move_to_a_pose_control.rst

```diff
@@ -13,34 +13,39 @@ Constructor
 ~~~~~~~~~~~
 
 .. code-block:: ipython3
+
    PathFinderController(Kp_rho, Kp_alpha, Kp_beta)
 
 Constructs an instantiate of the PathFinderController for navigating a 3-DOF wheeled robot on a 2D plane.
 
 Parameters:
-- Kp_rho: The linear velocity gain to translate the robot along a line towards the goal
-- Kp_alpha: The angular velocity gain to rotate the robot towards the goal
-- Kp_beta: The offset angular velocity gain accounting for smooth merging to the goal angle (i.e., it helps the robot heading to be parallel to the target angle.)
+
+- | :math:`Kp_rho` : The linear velocity gain to translate the robot along a line towards the goal
+- | :math:`Kp_alpha` : The angular velocity gain to rotate the robot towards the goal
+- | :math:`Kp_beta` : The offset angular velocity gain accounting for smooth merging to the goal angle (i.e., it helps the robot heading to be parallel to the target angle.)
```

Comment:
The math does not rendered properly, maybe you should `Kp_{rho}`? Please check other math renderings.
<img width="808" alt="SS 2021-12-26 at 10 42 46" src="https://user-images.githubusercontent.com/3813847/147396744-d19167f5-edea-41e8-a990-dbd1b4fcb24e.png">

Reply:
Interesting. What I have in the preview is different form yours. To test it, I just included {rho} in line 23. The picture below the subscripts with _{...} and without _{...}. If _{...} fixes it on the final online doc, I will apply it to all other subscripts too.

![image](https://user-images.githubusercontent.com/93126501/147413042-4ab29892-6cf7-4c76-959c-697cb5ec0189.png)

Reply:
Current code show this in my enviroment:
<img width="792" alt="SS 2021-12-27 at 19 45 38" src="https://user-images.githubusercontent.com/3813847/147464348-649a93ac-a9a0-4400-870d-7b3014fc6d26.png">
 Maybe, your environment has an issue about `sphinx.ext.mathjax' .

Reply:
I decided to make it simple and used bold text for the parameter definitions.